solr/solrj/src/java/org/apache/solr/client/solrj/io/stream/SolrStream.java

```diff
@@ -268,8 +275,7 @@ private Map mapFields(Map fields, Map<String,String> mappings) {
     return fields;
   }
 
-  // temporary...
-  public TupleStreamParser constructParser(SolrClient server, SolrParams requestParams) throws IOException, SolrServerException {
+  private TupleStreamParser constructParser(SolrParams requestParams) throws IOException, SolrServerException {
```

Comment:
Didn't seem like this method needed to be public and we already get a SolrClient in the open method, so no need to pass it. However, this breaks a public method signature, so is only for Solr 9.x and shouldn't be back-ported to 8.x